mycelium/src/tun/darwin.rs

```diff
@@ -174,6 +170,79 @@ fn validate_utun_name(input: &str) -> bool {
         .is_ok()
 }
 
+/// Validates the user-supplied TUN interface name
+///
+/// - If the name is valid and not in use, it will be the TUN name
+/// - If the name is valid but already in use, an error will be thrown
+/// - If the name is not valid, we try to find the first freely available TUN name
+fn find_available_utun_name(preferred_name: &str) -> Result<String, io::Error> {
+    println!("preferred name: {preferred_name}");
```

Comment:
This should be removed / replaced by a tracing compatible line (something like trace!/debug!)